contrib/hack/test-apiserver.sh

```diff
@@ -44,16 +44,19 @@ NO_TTY=1 kubectl config set-cluster service-catalog-cluster --server=https://${D
 set -x
 NO_TTY=1 kubectl create -f contrib/examples/apiserver/broker.yaml
 NO_TTY=1 kubectl create -f contrib/examples/apiserver/serviceclass.yaml
+NO_TTY=1 kubectl create -f contrib/examples/apiserver/serviceplan.yaml -v 9
```

Comment:
is the -v argument intentional here?

Reply:
debug

Reply:
now that it appears to be working, can we remove the -v ?

Reply:
yes. whoops.